hasura/graphql-engine
# console: support partitioned tables (close #5071)

close https://github.com/hasura/graphql-engine/issues/5071

### Description

How to test it?

1. Create partitioned  table:
```
CREATE TABLE measurement (
    logdate         date not null,
    peaktemp        int,
    unitsales       int
) PARTITION BY RANGE (logdate);

CREATE TABLE measurement_y2017 PARTITION OF measurement
  FOR VALUES FROM ('2017-01-01') TO ('2018-01-01');

CREATE TABLE measurement_y2018 PARTITION OF measurement 
  FOR VALUES FROM ('2018-01-01') TO ('2019-01-01');
```

2. All three tables should be visible in the console. 

### Changelog


### Affected components
